00-HTML-CSS-basics/html/toDo.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<title>To Do</title>
+		<meta name="viewport" content="width=device-width,initial-scale=1"/>
```

Comment:
**Mega-minor:** when using a self-closing tag, it would be good to leave an space before the slash ``<meta "..." />`` (just for code styling purposes...).